src/notify/subscription/db.go

```diff
@@ -14,26 +14,26 @@ var prepStmts struct {
 func CreateDB(db *sql.DB) error {
 	myDB = db
 
-	_, err := myDB.Exec(`
-              CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
-              CREATE TABLE IF NOT EXISTS subscriptions (
-                id UUID PRIMARY KEY,
-                event_type VARCHAR(64),
-                context VARCHAR(64),
-                account_id UUID,
-                created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP )
-              `)
-	if err != nil {
+	if _, err := myDB.Exec(`
```

Comment:
Just trying to follow the conventions I found in [the golang error blogpost](http://blog.golang.org/error-handling-and-go).

The article is 4 years old, so not sure if it is still the defacto pattern to toss the error-case-if logic around the executing function.